ansible/roles/fyrevm_provision/tasks/fyrevm_provision.yml

```diff
@@ -132,13 +226,23 @@
   shell: 'ssh-keyscan -H {{ emberFQDN }} >> ~/.ssh/known_hosts'
   when: "fyre_modifyKnownHosts|bool == true"
 
+- name: add_host info
+  debug:
+    msg: 
+    - "{{ emberFQDN }}"
+    - "{{ target_group }}"
+    - "{{ target_user|default('root') }}"
+    - "{{ clusterName }}"
+    - "{{ clusterName }}"
+    - "{{ fyre_addAnsibleHost }}"
+    verbosity: 3
+
 - add_host:
    name: "{{ emberFQDN }}"
    groups: "{{ target_group }}"
    ansible_user: "{{ target_user|default('root') }}"
    clusterName: "{{ clusterName }}"
    stackName: "{{ clusterName }}"
-  args: "{{ extra_props| default(omit) }}"
```

Comment:
extra_props allows a new host to be added with extra_props, this allows good modelling, Not sure on the reasoning behind this removal